ext/pdo/tests/gh8626.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+GH-8626: PDOStatement->execute() failed, then execute successfully, errorInfo() information is incorrect
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo')) die('skip');
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+$driver = substr(getenv('PDOTEST_DSN'), 0, strpos(getenv('PDOTEST_DSN'), ':'));
+if (!in_array($driver, array('mysql', 'pgsql', 'sqlite'))) {
```

Comment:
According to the documentation all drivers support it and it's the documented behaviour:

>If the SQLSTATE error code is not set or there is no driver-specific error, the elements following element 0 will be set to null.

Please remove that skip and make it applicable to all database drivers.